packages/public/frontend/serverless.js

```diff
@@ -16,47 +38,38 @@ const map = {
   '.png': 'image/png'
 }
 
-const server = new http.Server((req, res) => {
-  // parse URL and extract path
-  const pathname = url.parse(req.url).pathname
-
-  // render page
-  if (pathname === '/__footer') {
-    return footer.render(req, res)
-  }
-  if (pathname === '/__comments') {
-    return comments.render(req, res)
-  }
+// ONLY for development
+function serveAssets(pathname, req, res, next) {
+  if (process.env.SERVE_ASSETS === 'true') {
+    // based on the URL path, extract the file extention. e.g. .js, .doc, ...
+    const ext = path.parse(pathname).ext
```

Comment:
If possible, I'd rather have that not part of the production build (as it only is needed for dev). What do you think about an extra file `serverless.development.js` that imports the server from `serverless.js` and adds the additional middleware for routing?

Reply:
This sounds reasonable

Reply:
done